wormhole-connect/src/routes/sdkv2/route.ts

```diff
@@ -180,7 +180,9 @@ export class SDKv2Route {
       console.error(`Error thrown in isRouteAvailable`, e);
       // TODO is this the right place to try/catch these?
       // or deeper inside SDKv2Route?
-      return false;
+
+      // Re-throw for the caller to handle and surface the error message
+      throw e;
```

Comment:
@artursapek @kev1n-peters I need your input here. It seems `isRouteAvailable` function here eats up the exception, but we need that to surface the error. I first thought maybe we can return the error message instead of re-throwing; but that messes up the API signature (a method called isRouteAvailable returning a string error message, IDK 🤔). Do you see any issues with the re-throw? There is one more call to this in `routes/operator.ts`, do you think we need another try/catch there as well?

Reply:
I don't see an issue with re-throwing at the moment